src/components/Blog/Blog.react.js

```diff
@@ -206,6 +210,23 @@ class Blog extends Component {
     this.scrollToTop();
   };
 
+  showScrollToTop = () => {
+    let valScrolledFromOriginVertical = window.scrollY;
```

Comment:
How about directly using window.scrollY,
Do:
`window.scrollY >= 90 ? this.setState({showScrollTop: true}) : this.setState({showScrollTop: false})`

Reply:
Instead do this 😄 
```
this.setState({
   showScrollToTop : window.scrollY >= 90 ? true : false
});
```